handlers/bounty_test.go

```diff
@@ -1793,7 +1801,7 @@ func TestPollInvoice(t *testing.T) {
 		ro.Post("/poll/invoice/{paymentRequest}", bHandler.PollInvoice)
 
 		rr := httptest.NewRecorder()
-		req, err := http.NewRequestWithContext(authorizedCtx, http.MethodPost, "/poll/invoice/1", bytes.NewBufferString(`{}`))
```

Comment:
@AbdulWahab3181 you didn't assert that the Bounty was set to `Paid: True`, and the invoice was set to `Status: true`.

Reply:
@elraphty Addressed